graph/pull.go

```diff
@@ -567,11 +585,13 @@ func (s *TagStore) pullV2Tag(r *registry.Session, out io.Writer, endpoint *regis
 						Action:    "Extracting",
 					}))
 				if err != nil {
+					out.Write(sf.FormatProgress(stringid.TruncateID(d.img.ID), "Problem extracting", nil))
```

Comment:
Same as above error message, use string value from error... and probably should be written by the caller since the error is being returned.